### ISSUE TITLE
[QA] GUI tests pipeline is taking significant time to complete - investigate

### ISSUE THREAD

**Issue by saw-jan** [owncloud/client#11179]
### Describe the QA issue

GUI tests pipeline in branch 5 is taking significant time to complete - investigate

oCIS server: >35 mins

![Screenshot from 2023-09-22 10-07-22](https://github.com/owncloud/client/assets/52366632/cdada2c6-6dd4-44d5-bb57-935bb0a6c707)

Examples:
https://drone.owncloud.com/owncloud/client/16503
https://drone.owncloud.com/owncloud/client/16490
https://drone.owncloud.com/owncloud/client/16500

**Comment by saw-jan** [comment]
Sync pattern (sync messages pattern) for syncing account that _doesn't have any files_ in the server has changed.
Pattern was changed along with PR https://github.com/owncloud/client/pull/11153

Before:
```bash
STATUS:OK:/tmp/client-bdd/Alice/
STATUS:OK:/tmp/client-bdd/Alice/
REGISTER_PATH:/tmp/client-bdd/Alice
STATUS:OK:/tmp/client-bdd/Alice
UPDATE_VIEW:/tmp/client-bdd/Alice
STATUS:OK:/tmp/client-bdd/Alice
STATUS:OK:/tmp/client-bdd/Alice
STATUS:OK:/tmp/client-bdd/Alice
UPDATE_VIEW:/tmp/client-bdd/Alice
```
Current:
```bash
STATUS:OK:/tmp/client-bdd/Alice/
STATUS:OK:/tmp/client-bdd/Alice/
STATUS:OK:/tmp/client-bdd/Alice
STATUS:OK:/tmp/client-bdd/Alice
REGISTER_PATH:/tmp/client-bdd/Alice
```

Current sync messages don't contain `UPDATE_VIEW` message

**Comment by saw-jan** [comment]
CC @TheOneRing

**Comment by saw-jan** [comment]
Maybe fixed by PR #11202

**Comment by saw-jan** [comment]
Closing as fixed

![Screenshot from 2023-09-22 10-08-05](https://github.com/owncloud/client/assets/52366632/261a88a5-75fa-4ea0-8e1a-4e6a6b8b54a3)
